tests/io/test_from_geopandas.py

```diff
@@ -13,21 +13,23 @@ def test_read_positionfixes_gpd(self):
         """Test if the results of reading from gpd and csv agrees."""
         gdf = gpd.read_file(os.path.join("tests", "data", "positionfixes.geojson"))
         gdf.set_index("id", inplace=True)
-        pfs_from_gpd = ti.io.from_geopandas.read_positionfixes_gpd(gdf, user_id="User", geom="geometry", tz="utc")
+        pfs_from_gpd = ti.io.from_geopandas.read_positionfixes_gpd(gdf, user_id="User", geom_col="geometry", tz="utc")
```

Comment:
In `read_positionfixes.gpd` there is an optional parameter `columns`. I thought I could use this parameter to rename the column to `geometry`, but it looks like all columns can be renamed except for the `geometry` column. Therefore, I had to rename the column in the test function in order to make `assert_frame_equal` work.